tests/main/views/test_services.py

```diff
@@ -305,6 +305,20 @@ def test_pre_live_framework_causes_404(self):
 
         assert res.status_code == 404
 
+    def test_service_on_framework_without_a_display_service_manifest_causes_404(self):
+        self.service = self._get_g6_service_fixture_data()
+        self.service['services']['frameworkSlug'] = 'digital-outcomes-and-specialists'
+        data_api_client.get_service.return_value = self.service
+        data_api_client.get_framework.return_value = self._get_framework_fixture_data(
+            'digital-outcomes-and-specialists'
+        )
+
+        service_id = self.service['services']['id']
+
+        res = self.client.get('/g-cloud/services/{}'.format(service_id))
```

Comment:
I'm a little confused about what's going on here - you're getting g-cloud (6) service data, setting the `frameworkSlug` to `digital-outcomes-and-specialists` and then accessing the url `'/g-cloud/services/{}'`?